ci/unit_tests.bat

```diff
@@ -0,0 +1,41 @@
+@echo off
```

Comment:
No changes here ... just a copy/paste from the old name (but kept the old name in there since it is currently in use)

Reply:
If it's a copy/paste I think maybe one of them should just execute the other

Reply:
Once I get the changes pushed to Jenkins, I will delete all the old scripts.  The duplicates should only be in source control for a day or two.